news/2 Fixes/10752.md

```diff
@@ -0,0 +1 @@
+Stopped using localization in certain phrases that would break some of the extension features while we investigate the underlying reason.
```

Comment:
```suggestion
Temporarily disable localising certain phrases that would break some of the extension features while we investigate the underlying reason.
```